frontend/src/prompted_readings/analysis_view.js

```diff
@@ -1,4 +1,22 @@
 import React from "react";
+import PropTypes from "prop-types";
+
+function SentimentScores(props) {
```

Comment:
tagging @ryaanahmed -- I think that we've decided to create classes for all React-widgets instead of using capital function names in order to avoid confusion.

Reply:
So we can replace this was a class that extends React.Component.

Reply:
Yes -- but haven't communicated this to the team, and I don't think there's a way to lint for it. Will have a think and just start spreading the word.